Sources/BraintreePaymentFlow/BTLocalPaymentRequest.swift

```diff
@@ -63,6 +63,9 @@ import BraintreeDataCollector
     var paymentID: String?
     weak var paymentFlowClientDelegate: BTPaymentFlowClientDelegate?
     var correlationID: String?
+    var paymentFlowAPIClient: BTAPIClient? {
```

Comment:
nit: can we group these properties together? It looks like in other files we would do vars > computed vars > weak vars.

Reply:
Not sure what you mean by group the properties?

Reply:
Right now we have:
```swift
var paymentID: String?
weak var paymentFlowClientDelegate: BTPaymentFlowClientDelegate?
var correlationID: String?
var paymentFlowAPIClient: BTAPIClient? { ... }
```

Should be:
```swift
var paymentID: String?
var correlationID: String?

var paymentFlowAPIClient: BTAPIClient? { ... }

weak var paymentFlowClientDelegate: BTPaymentFlowClientDelegate?
```

Reply:
Done